### ISSUE TITLE
Test error : memory descriptor

### ISSUE THREAD

**Issue by nassimaNoufail** [ZJULearning/pixel_link#98]
During the test, the following error has occurred, I don't know how to solve this problem, can you help me with it?
AbortedError (see above for traceback): Operation received an exception:Status: 3, message: could not initialize a memory descriptor, in file tensorflow/core/kernels/mkl_softmax_op.cc:164
	 [[node test/Softmax (defined at /root/PycharmProjects/pixel_link/nets/pixel_link_symbol.py:177)  = _MklSoftmax[T=DT_FLOAT, _kernel="MklOp", _device="/job:localhost/replica:0/task:0/device:CPU:0"](test/pixel_cls/add_3, test/pixel_cls/add_3:1)]]

**Comment by Dgs29** [comment]
are you doing this for Document text images?
